src/transpiler.js

```diff
@@ -83,3 +94,91 @@ function getGivenName({ names = undefined }) {
 function getFamilyName({ names = undefined }) {
   return names && names[0] && names[0].familyName
 }
+
+// cozy -> google
+function getNames({ name = undefined }) {
+  return [name]
+}
+
+function getEmailAddresses({ email = [] }) {
+  return email.map(email => ({
+    metadata: {
+      primary: email.primary
+    },
+    type: email.type,
+    value: email.address
+  }))
+}
+
+function getPhoneNumbers({ phone = [] }) {
+  return phone.map(
+    phoneItem =>
+      phoneItem && {
+        value: phoneItem.number,
+        metadata: {
+          primary: phoneItem.primary || false
+        },
+        type: phoneItem.type,
+        formattedType: phoneItem.label
+      }
+  )
+}
+
+function getAddresses({ address = [] }) {
+  return address.map(
+    addressItem =>
+      addressItem && {
+        city: addressItem.city,
+        country: addressItem.country,
+        postalCode: addressItem.postcode,
+        streetAddress: addressItem.street,
+        metadata: {
+          primary: addressItem.primary || false
+        },
+        type: addressItem.type,
+        formattedType: addressItem.label,
+        formattedValue: addressItem.formattedAddress
+      }
+  )
+}
+
+function getBirthdays({ birthday = [] }) {
+  if (birthday) {
```

Comment:
The readibility can be simplified by removing the if/else block:
If `birthday` is set by default to `[]`, then `if (birthday)` will always be triggered.
Then the `if` statement is useless.

Reply:
in fact birthday is a string. I change the default value to `null` and edit the code